aws/credentials/ssocreds/provider_test.go

```diff
@@ -131,6 +147,82 @@ func TestProvider(t *testing.T) {
 			},
 			ExpectedExpire: time.Date(2021, 01, 20, 21, 22, 23, 0.123e9, time.UTC),
 		},
+		"custom cached token file": {
```

Comment:
i tried debugging this myself for a little bit, but couldnt find the answer. the referenced test data in `custom_cached_token.json` has an access token that is expired. yet, somehow it doesnt return an expired error. i ran it through the debugger and saw that the unmarshaler (`func (r *rfc3339) UnmarshalJSON` used by `loadTokenFile`) and in the custom is returning a date way in the future (like year 3900) and not the the year specified in the `expiresAt` field which is 2021. do you know whats happening here?

Reply:
discussed this offline, i was calculating the epoch time with wrong relative date (january 1 1970). when the unmarshaller is using january 1 1 as the relative date. so no issue here